cmd/fip-controller/main.go

```diff
@@ -28,6 +28,8 @@ func main() {
 	flag.StringVar(&controllerConfig.NodeName, "node-name", "", "Kubernetes Node name")
 	flag.StringVar(&controllerConfig.PodName, "pod-name", "", "Kubernetes pod name")
 	flag.StringVar(&controllerConfig.LogLevel, "log-level", "Info", "Log level")
+	flag.StringVar(&controllerConfig.FloatingIPLabelSelector, "floating-ip-label-selector", "", "Selector for Floating IPs")
+	flag.StringVar(&controllerConfig.NodeLabelSelector, "node-label-selector", "", "Selector for Nodes")
```

Comment:
I thought about this as well but haven't found an easy way to find all nodes a deployment is running on.

If we really want this feature the only way I found to possibly do this is by labeling the deployment/daemonset to find it from inside the pod and then getting all hostIPs from all pods inside the deployment/daemonset. 
I am not completely sure how the hostIP on a pod is assigned but this will either not work or will remove the need to specify wether to have internal or external node ips.

Reply:
This is now possible with the PodLabelSelector